Decred Wallet/Features/Custom Dialogs/SimpleTextInputDialog.swift

```diff
@@ -93,7 +93,7 @@ class SimpleTextInputDialog: UIViewController {
     }
     
     @objc func textFieldEditingChanged() {
-        self.submitButton.isEnabled = self.verifyInput ? self.currentValue == self.textField.text : (self.textField.text ?? "").count > 0
+        self.submitButton.isEnabled = self.verifyInput ? self.currentValue == self.textField.text : (self.textField.text?.trimmingCharacters(in: .whitespaces) ?? "").count > 0
```

Comment:
The trimming is only during verification which allows the actual text to be saved with white spaces.  

<img src="https://user-images.githubusercontent.com/19960200/120068112-f8562080-c076-11eb-86e6-e3d47f37302a.jpg" width="300">